scib/metrics/lisi.py

```diff
@@ -692,8 +694,7 @@ def convertToOneHot(vector, num_classes=None):
     return result.astype(int)
 
 
-# DEPRECATED
-# This code scales clisi incorrectly!
+@deprecated
```

Comment:
Is there a reason to keep this or can it be deleted?

Reply:
This is only used in the `scib.me.lisi.lisi` function, but we use `scib.me.lisi.lisi_graph` where scaling is replaced with newer code. Can we also consider `scib.me.lisi.lisi` as deprecated, since we aren't using it anywhere (I marked it as such)?